Plugin/Controller/Adminhtml/Integration/SavePlugin.php

```diff
@@ -116,11 +147,63 @@ private function disableAllStoreAssociations($integrationId)
     {
         $storeIntegrationCollection = $this->storeIntegrationCollection->create();
         $storeIntegrations = $storeIntegrationCollection
-            ->addFieldToFilter(\Extend\Integration\Api\Data\StoreIntegrationInterface::INTEGRATION_ID, $integrationId)
+            ->addFieldToFilter(
+                \Extend\Integration\Api\Data\StoreIntegrationInterface::INTEGRATION_ID,
+                $integrationId
+            )
             ->load();
         foreach ($storeIntegrations->getItems() as $storeIntegration) {
             $storeIntegration->setDisabled(1);
             $this->storeIntegrationResource->save($storeIntegration);
         }
     }
+
+    /**
+     * Send stores to Extend Magento Service when added to an integration after it's already been activated.
+     *
+     * @param $integrationId
+     * @param $storeId
+     * @return void
+     * @throws IntegrationException
+     * @throws LocalizedException
+     * @throws NoSuchEntityException
+     * @throws Exception
+     */
+    private function sendIntegrationToExtend($integrationId, $storeId)
+    {
+        $integrationStore = $this->integrationStoresRepository->getByStoreIdAndIntegrationId(
+            $storeId,
+            $integrationId
+        );
+        $integration = $this->integrationService->get($integrationId);
+        $oauth = $this->oauthService->loadConsumer($integration->getConsumerId());
+        $store = $this->storeManager->getStore($storeId);
+
+        $endpoint = [
+            'path' => IntegrationService::EXTEND_INTEGRATION_ENDPOINTS['webhooks_orders_create'],
```

Comment:
Should this be `webhooks_stores_create`?

Reply:
good catch, updated now